pkg/routing/redisrouter.go

```diff
@@ -161,7 +161,7 @@ func (r *RedisRouter) StartParticipantSignal(ctx context.Context, roomName livek
 		return
 	}
 
-	return r.StartParticipantSignalWithNodeID(ctx, roomName, pi, livekit.NodeID(rtcNode.Id))
+	return r.StartParticipantSignalWithNodeID(ctx, roomName, pi, livekit.NodeID(rtcNode.Id), "selector")
```

Comment:
Just marking it as "selector" as the `selectionReason` for now. In the future, can change `Selector` interface `SelectNode` method to return a proper reason and use that.